include/vara/Feature/FeatureSort.h

```diff
@@ -0,0 +1,40 @@
+#ifndef VARA_FEATURE_FEATURESORT_H
+#define VARA_FEATURE_FEATURESORT_H
+
+#include "FeatureModel.h"
+
+namespace vara::feature {
+
+class FeatureSort {
```

Comment:
Currently, there are insertion methods missing. And I also do not see how this is used in the FM. I'll let you finish the impl before I'll continue to look at stuff :)